src/table/utils/__tests__/selections-util.spec.ts

```diff
@@ -217,21 +237,12 @@ describe('selections-utils', () => {
       const newState = reducer(state, action);
       expect(newState).toEqual(state);
     });
-
-    it('should return error when incorrect action type', () => {
```

Comment:
Not sure if this is needed anymore. There can't be an incorrect action since it would mean the wrong type and not even compile

Reply:
Correct!